inc/common/purge.php

```diff
@@ -1,6 +1,6 @@
 <?php
 defined( 'ABSPATH' ) || die( 'Cheatin&#8217; uh?' );
-
+use WP_Rocket\Logger\Logger;
```

Comment:
Is the Logger used in this file currently?

Reply:
It doesn’t seem so.

Reply:
Nope. Forgot to remove it.